scripts/common/tests/test.js

```diff
@@ -138,16 +138,17 @@ export class WNGTest {
     if (this.actor.hasCondition("dying"))
     {
       this.result.isWrathCritical = this.result.dice.every(r => r.isWrath && r.result === 6);
-      this.result.gainTraumaticInjury = this.result.isWrathComplication 
+      this.result.gainTraumaticInjury = this.result.isWrathComplication
+      return;
     }
-    else 
-      this.result.isWrathCritical = this.result.dice.some(r => r.isWrath && r.result === 6);
```

Comment:
Why was this else removed? Without it, the logic concerning criticals when dying gets overwritten.

Reply:
It isn't removed, the logic is just better. The `if` is fine, but the `else` isn't needed as you return in the `if`